docs/developer_guidelines.md

```diff
@@ -0,0 +1,91 @@
+# Developer guidelines
+
+## Table of Contents 📜
+
+<!-- vim-markdown-toc GFM -->
+
+* [Commits 🪢](#commits-)
+    * [Message style](#message-style)
+    * [Message content](#message-content)
+* [Documentation in source code 📑](#documentation-in-source-code-)
+* [Coding standards 📚](#coding-standards-)
+    * [C/C++ 🇨](#cc-)
+    * [Python 🐍](#python-)
+    * [Rust 🦀](#rust-)
+    * [JavaScript/TypeScript 🇯🇸 / 🇹🇸](#javascripttypescript---)
+
+<!-- vim-markdown-toc -->
```

Comment:
![image](https://user-images.githubusercontent.com/26332685/168969036-9b5b0737-e972-41e0-83c2-f692c883f24a.png)

The `C` emoji is barely visible, 
Google chrome v100.0.4896.127 doesnt display the `JS/TS` emojis

Reply:
Firefox 100.0 does not have any problems displaying it.